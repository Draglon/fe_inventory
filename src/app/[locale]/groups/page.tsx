export { default } from "@/views/Groups";
