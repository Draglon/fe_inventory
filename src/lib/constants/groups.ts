export const GROUPS = [
  {
    id: 1,
    title: "Длинное предлинное длиннючее название прихода",
    date: "2017-06-29 12:09:33",
    quantity: 23,
    description: "Продукта",
    products: [
      {
        id: "1",
        serialNumber: "SN-12.3456789",
        isNew: 1,
        photo: "/images/products/monitors/product.webp",
        title: "Product 1",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "Свободен",
        group: "Длинное предлинное длиннючее название группы",
        fullName: null,
        parish: "Длинное предлинное длиннючее название прихода",
      },
      {
        id: "2",
        serialNumber: "SN-12.3456789",
        isNew: 0,
        photo: "/images/products/monitors/product.webp",
        title: "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "Свободен",
        group: "Длинное предлинное длиннючее название группы",
        fullName: "John Smith",
        parish: "Длинное предлинное длиннючее название прихода",
      },
      {
        id: "3",
        serialNumber: "SN-12.3456789",
        isNew: 0,
        photo: "/images/products/monitors/product.webp",
        title: "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "В ремонте",
        group: "Длинное предлинное длиннючее название группы",
        fullName: null,
        parish: "Длинное предлинное длиннючее название прихода",
      },
    ],
  },

  {
    id: 2,
    title: "Длинное предлинное длиннючее название прихода",
    date: "2017-06-29 12:09:33",
    quantity: 23,
    description: "Продукта",
    products: [
      {
        id: "1",
        serialNumber: "SN-12.3456789",
        isNew: 1,
        photo: "/images/products/monitors/product.webp",
        title: "Product 1",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "В наличии",
        group: "Длинное предлинное длиннючее название группы",
        fullName: null,
        parish: "Длинное предлинное длиннючее название прихода",
      },
      {
        id: "2",
        serialNumber: "SN-12.3456789",
        isNew: 0,
        photo: "/images/products/monitors/product.webp",
        title: "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "В ремонте",
        group: "Длинное предлинное длиннючее название группы",
        fullName: "John Smith",
        parish: "Длинное предлинное длиннючее название прихода",
      },
      {
        id: "3",
        serialNumber: "SN-12.3456789",
        isNew: 0,
        photo: "/images/products/monitors/product.webp",
        title: "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "Свободен",
        group: "Длинное предлинное длиннючее название группы",
        fullName: null,
        parish: "Длинное предлинное длиннючее название прихода",
      },
    ],
  },

  {
    id: 3,
    title: "Длинное предлинное длиннючее название прихода",
    date: "2017-06-29 12:09:33",
    quantity: 23,
    description: "Продукта",
    products: [
      {
        id: "1",
        serialNumber: "SN-12.3456789",
        isNew: 1,
        photo: "/images/products/monitors/product.webp",
        title: "Product 1",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "Свободен",
        group: "Длинное предлинное длиннючее название группы",
        fullName: null,
        parish: "Длинное предлинное длиннючее название прихода",
      },
      {
        id: "2",
        serialNumber: "SN-12.3456789",
        isNew: 0,
        photo: "/images/products/monitors/product.webp",
        title: "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "В ремонте",
        group: "Длинное предлинное длиннючее название группы",
        fullName: "John Smith",
        parish: "Длинное предлинное длиннючее название прихода",
      },
      {
        id: "3",
        serialNumber: "SN-12.3456789",
        isNew: 0,
        photo: "/images/products/monitors/product.webp",
        title: "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
        type: "monitors",
        specification: "Specification 1",
        guarantee: {
          start: "2017-06-29 12:09:33",
          end: "2017-06-29 12:09:33",
        },
        price: [
          { value: 100, symbol: "USD", isDefault: 0 },
          { value: 2600, symbol: "UAH", isDefault: 1 },
        ],
        order: 1,
        date: "2017-06-29 12:09:33",
        status: "В ремонте",
        group: "Длинное предлинное длиннючее название группы",
        fullName: null,
        parish: "Длинное предлинное длиннючее название прихода",
      },
    ],
  },
];
