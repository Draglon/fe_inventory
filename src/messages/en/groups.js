const groups = {
  "Groups": {
    "layout": {
      "title": "Groups",
      "description": "Groups - description",
    },
    "title": "Groups",
    "addProduct": "Add product",
  },
};

export default groups;