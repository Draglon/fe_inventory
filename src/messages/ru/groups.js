const groups = {
  "Groups": {
    "layout": {
      "title": "Группы",
      "description": "Группы - описание",
    },
    "title": "Группы",
  },
};

export default groups;