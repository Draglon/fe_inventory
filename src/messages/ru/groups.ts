const groups = {
  "Groups": {
    "layout": {
      "title": "Группы",
      "description": "Группы - описание",
    },
    "title": "Группы",
    "addProduct": "Добавить продукт",
  },
};

export default groups;