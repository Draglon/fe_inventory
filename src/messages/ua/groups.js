const groups = {
  "Groups": {
    "layout": {
      "title": "Групи",
      "description": "Групи - опис",
    },
    "title": "Групи",
  },
};

export default groups;