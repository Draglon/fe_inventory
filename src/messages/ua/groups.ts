const groups = {
  "Groups": {
    "layout": {
      "title": "Групи",
      "description": "Групи - опис",
    },
    "title": "Групи",
    "addProduct": "Додати продукт",
  },
};

export default groups;