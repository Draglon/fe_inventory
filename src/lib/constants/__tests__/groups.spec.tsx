import * as groups from "../groups";

it("groups constants match snapshot", () => {
  expect(groups).toMatchSnapshot();
});
