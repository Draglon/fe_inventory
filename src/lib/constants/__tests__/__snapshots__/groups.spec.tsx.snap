// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`groups constants match snapshot 1`] = `
{
  "GROUPS": [
    {
      "date": "2017-06-29 12:09:33",
      "description": "Продукта",
      "id": 1,
      "products": [
        {
          "date": "2017-06-29 12:09:33",
          "fullName": null,
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "1",
          "isNew": 1,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "Свободен",
          "title": "Product 1",
          "type": "monitors",
        },
        {
          "date": "2017-06-29 12:09:33",
          "fullName": "John Smith",
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "2",
          "isNew": 0,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "Свободен",
          "title": "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
          "type": "monitors",
        },
        {
          "date": "2017-06-29 12:09:33",
          "fullName": null,
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "3",
          "isNew": 0,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "В ремонте",
          "title": "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
          "type": "monitors",
        },
      ],
      "quantity": 23,
      "title": "Длинное предлинное длиннючее название прихода",
    },
    {
      "date": "2017-06-29 12:09:33",
      "description": "Продукта",
      "id": 2,
      "products": [
        {
          "date": "2017-06-29 12:09:33",
          "fullName": null,
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "1",
          "isNew": 1,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "В наличии",
          "title": "Product 1",
          "type": "monitors",
        },
        {
          "date": "2017-06-29 12:09:33",
          "fullName": "John Smith",
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "2",
          "isNew": 0,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "В ремонте",
          "title": "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
          "type": "monitors",
        },
        {
          "date": "2017-06-29 12:09:33",
          "fullName": null,
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "3",
          "isNew": 0,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "Свободен",
          "title": "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
          "type": "monitors",
        },
      ],
      "quantity": 23,
      "title": "Длинное предлинное длиннючее название прихода",
    },
    {
      "date": "2017-06-29 12:09:33",
      "description": "Продукта",
      "id": 3,
      "products": [
        {
          "date": "2017-06-29 12:09:33",
          "fullName": null,
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "1",
          "isNew": 1,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "Свободен",
          "title": "Product 1",
          "type": "monitors",
        },
        {
          "date": "2017-06-29 12:09:33",
          "fullName": "John Smith",
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "2",
          "isNew": 0,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "В ремонте",
          "title": "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
          "type": "monitors",
        },
        {
          "date": "2017-06-29 12:09:33",
          "fullName": null,
          "group": "Длинное предлинное длиннючее название группы",
          "guarantee": {
            "end": "2017-06-29 12:09:33",
            "start": "2017-06-29 12:09:33",
          },
          "id": "3",
          "isNew": 0,
          "order": 1,
          "parish": "Длинное предлинное длиннючее название прихода",
          "photo": {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 40,
            "src": "/img.jpg",
            "width": 40,
          },
          "price": [
            {
              "isDefault": 0,
              "symbol": "USD",
              "value": 100,
            },
            {
              "isDefault": 1,
              "symbol": "UAH",
              "value": 2600,
            },
          ],
          "serialNumber": "SN-12.3456789",
          "specification": "Specification 1",
          "status": "В ремонте",
          "title": "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
          "type": "monitors",
        },
      ],
      "quantity": 23,
      "title": "Длинное предлинное длиннючее название прихода",
    },
  ],
}
`;
